### DISCUSSION TITLE
Setting for the default position of plugin UI on server screen

### DISCUSSION THREAD

**Discussion by th3funnychurch3s** [apohl79/audiogridder#1201] [Ideas]
I use a touchscreen on the computer running the AG server to control the plugins, as well as Touch OSC to control other DAW functions. (Naturally I don't use the screen capture mode). I think a nice feature would be to set the default position of the plugin UI as they pop up, i.e. top-left or bottom centre, middle, etc. In this way the plugin windows could be within easier reach (let's say bottom-middle for bigger touch screen devices) and it would enable you to sort of design other features around it. (think MP MIDI controller-style. I don't know how this would affect the screen capturing mode though?

**Comment by apohl79** [comment]
Understood, will create a feature request for this. Cheers.

**Comment by dmuckala** [comment]
I would love this as well!
